File: Prepare/Algorithms/Warmup/3-Compare-the-Triplets/Python3/main.py
alice = [int(x) for x in input().split(' ')]
bob = [int(x) for x in input().split(' ')]
alice_score, bob_score = 0, 0

assert len(alice) == len(bob)

for i in range(len(alice)):
    if alice[i] > bob[i]:
        alice_score += 1
    elif alice[i] < bob[i]:
        bob_score += 1
        
print(f"{alice_score} {bob_score}")

File: Prepare/Algorithms/Warmup/3-Compare-the-Triplets/C/main.c
#include <stdio.h>

int main() {
    int alice[3];
    for (int i = 0; i < 3; i++) {
        scanf("%d", &alice[i]);
    }

    int bob[3];
    for (int i = 0; i < 3; i++) {
        scanf("%d", &bob[i]);
    }

    int alice_score = 0;
    int bob_score = 0;

    for (int i = 0; i < 3; i++) {
        if (alice[i] > bob[i]) {
            alice_score++;
        } else if (alice[i] < bob[i]) {
            bob_score++;
        }
    }

    printf("%d %d", alice_score, bob_score);
    
    return 0;
}